activerecord/test/cases/validations/uniqueness_validation_test.rb

```diff
@@ -86,6 +99,19 @@ def test_validates_uniqueness_with_validates
     assert t2.errors[:title]
   end
 
+  if current_adapter? :PostgreSQLAdapter
+    def test_validate_uniqueness_when_integer_out_of_range
+      entry = BigIntTest.create(four_bytes_int: (BigIntTest::PG_MAX_INTEGER + 1))
+      assert entry.errors[:four_bytes_int] , ['is not included in the list']
```

Comment:
This should be `assert_equals`

Please remove the leading space before the comma.